docs/topics/email.txt

```diff
@@ -569,11 +569,11 @@ can :ref:`write your own email backend <topic-custom-email-backend>`.
 SMTP backend
 ~~~~~~~~~~~~
 
-.. class:: backends.smtp.EmailBackend(host=None, port=None, username=None, password=None, use_tls=None, fail_silently=False, use_ssl=None, timeout=None, ssl_keyfile=None, ssl_certfile=None, **kwargs)
+.. class:: backends.smtp.EmailBackend(host=None, port=None, username=None, password=None, use_tls=None, fail_silently=False, use_ssl=None, timeout=None, ssl_keyfile=None, ssl_certfile=None, ssl_cafile=None, **kwargs)
```

Comment:
This need a version changed note, just above `.. _topic-email-console-backend`
```
    .. versionchanged:: 5.2

        The ``ssl_cafile`` argument was added.
```

I suppose we should also add what `ssl_cafile` is?